packages/core/bootstrap/src/lib/cache-warmer/config.ts

```diff
@@ -34,7 +35,7 @@ export function get(): Config {
   return {
     hashOpts: getHashOpts(),
     unhealthyThreshold: Number(process.env.WARMUP_UNHEALTHY_THRESHOLD) || 3,
-    warmupInterval: Number(process.env.CACHE_MIN_AGE) || MINIMUM_AGE,
+    warmupInterval: Number(process.env.WARMUP_INTERVAL),
```

Comment:
This variable seems to not have been used at all prior to this PR. Hence, making it return `WARMUP_INTERVAL` or undefined should be fine.